apps/src/javalab/Playground.js

```diff
@@ -1,14 +1,39 @@
 import {PlaygroundSignalType} from './constants';
+import {assets, starterAssets} from '@cdo/apps/clientApi';
 
 export default class Playground {
-  constructor(onOutputMessage, onNewlineMessage, onJavabuilderMessage) {
+  constructor(
+    onOutputMessage,
+    onNewlineMessage,
+    onJavabuilderMessage,
+    levelName,
+    starterAssetsApi,
+    assetsApi
+  ) {
     this.onOutputMessage = onOutputMessage;
     this.onNewlineMessage = onNewlineMessage;
     this.onJavabuilderMessage = onJavabuilderMessage;
     this.isGameRunning = false;
     this.isGameOver = false;
+    this.levelName = levelName;
+    this.starterAssetFilenames = [];
+    this.starterAssetsApi = starterAssetsApi || starterAssets;
```

Comment:
Are these two assignments just to support tests? Maybe add comment if so?

Reply:
Yep they're just for testing - I'll add a comment